tools/depends/native/flatbuffers-native/Makefile

```diff
@@ -0,0 +1,60 @@
+include ../../Makefile.include
+PREFIX=$(NATIVEPREFIX)
+PLATFORM=$(NATIVEPLATFORM)
+DEPS=../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=flatbuffers
+VERSION=1.9.0
+SOURCE=v$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+BASE_URL=https://github.com/google/flatbuffers/archive
```

Comment:
Has to be uploaded to the mirrors.

Reply:
Can you upload https://github.com/google/flatbuffers/archive/v1.9.0.tar.gz (as `flatbuffers-1.9.0.tar.gz`) to the mirrors?